cmd/libvirt-provider/app/app.go

```diff
@@ -148,6 +153,10 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&o.GCVMGracefulShutdownTimeout, "gc-vm-graceful-shutdown-timeout", 5*time.Minute, "Duration to wait for the VM to gracefully shut down. If the VM does not shut down within this period, it will be forcibly destroyed by garbage collector.")
 	fs.DurationVar(&o.ResyncIntervalGarbageCollector, "gc-resync-interval", 1*time.Minute, "Interval for resynchronizing the garbage collector.")
 
+	fs.IntVar(&o.MaximumMachineEvents, "maximum-machine-events", 100, "Maximum number of machine events that can be stored.")
```

Comment:
i recommend keep naming convention general -> specific. machine-event-max-events
Same i recommend do for names of vars.

Reply:
applied.